gratipay/models/exchange_route.py

```diff
@@ -53,12 +49,14 @@ def from_address(cls, participant, network, address):
         return r
 
     @classmethod
-    def insert(cls, participant, network, address, error='', fee_cap=None, cursor=None):
+    def insert(cls, participant, network, address, error='', fee_cap=None, cursor=None,
+                                                                                 is_deleted=False):
```

Comment:
In fact, we should only be accepting `participant`, `network`, `address` and `fee_cap` when creating an exchange route. Routes are never created with errors, and routes are never created with `is_deleted=true`